build.gradle.kts

```diff
@@ -328,7 +328,7 @@ tasks.register("pythonDockerBuildPreCommit") {
 
 tasks.register("pythonLintPreCommit") {
   // TODO(https://github.com/apache/beam/issues/20209): Find a better way to specify lint and formatter tasks without hardcoding py version.
-  dependsOn(":sdks:python:test-suites:tox:py37:lint")
```

Comment:
we should lint against the lower version of python that we support. we should find how to exclude the file or portion of the code (perhaps by moving it to a different file), for the earlier version of the linter if necessary